src/repr/repr.ml

```diff
@@ -19,5 +19,6 @@ include Type
 
 (** {1 Miscellaneous modules} *)
 
+module Binary_codec = Binary_codec
```

Comment:
Is this a permanent or temporary solution to expose this feature?

Reply:
As discussed, the plan here was to expose some sort of binary namespace to provide direct access to the serialisation format. This could be used e.g. [here](https://github.com/mirage/irmin/blob/main/src/irmin-pack/pack_index.ml#L39-L41) where we're implicitly referencing the Repr codec (but avoiding using generics for performance reasons); the varint case is just a particularly awkward instance of this pattern.